docs/source/task_guides/int8-asr.mdx

```diff
@@ -0,0 +1,376 @@
+# int8 training for automatic speech recognition
+
+Quantization reduces the precision of floating point data types, reducing the memory required to store model weights. However, quantization degrades inference performance because you're losing information when you reduce the precision. 8-bit or `int8` quantization uses only a quarter precision, but it does not degrade performance because it doesn't just drop the bits or data. Instead, `int8` quantization *rounds* from one data type to another.
```

Comment:
```suggestion
Quantization reduces the precision of floating point data types, reducing the memory required to store model weights. However, quantization degrades inference performance because you lose information when you reduce the precision. 8-bit or `int8` quantization uses only a quarter precision, but it does not degrade performance because it doesn't just drop the bits or data. Instead, `int8` quantization *rounds* from one data type to another.
```